docs/content/kv_commands.md

```diff
@@ -101,26 +113,41 @@ yes
 
 ### `list`
 
-Outputs a list of all KV namespaces associated with your account id.
+Outputs a list of all keys in a given namespace. Optional params include
+1. `--prefix`: A prefix to filter listed keys
 
 #### Usage
+The example below uses Python's JSON pretty-printing command line tool to pretty-print output.
 
 ```sh
-$ wrangler kv:key list f7b02e7fc70443149ac906dd81ec1791 --prefix="public"
-🌀  Retrieving keys
-✨  Success:
-+------------------+----------------------------------+
-| KEY              | EXPIRATION                       |
-+------------------+----------------------------------+
-| "key"            | Wed Aug 28 10:28:44 CDT 2019     |
-+------------------+----------------------------------+
+$ wrangler kv:key list f7b02e7fc70443149ac906dd81ec1791 --prefix="public" | python -m json.tool
+[
+    {
+        "name": "public_key"
+    }, 
+    {
+        "name": "public_key_with_expiration",
+        "expiration": 1568014518
+    } 
+]
```

Comment:
So you removed the table for JSON output? I think if you prettier print it from wrangler it should be readable enough :+1:

Reply:
Yeah, the table would get super unwieldy for very large namespaces :D JSON is easier for folks to handle.